src/screens/SignInV2/elements/BottomOverlay/index.js

```diff
@@ -50,7 +51,7 @@ const BottomOverlay = ({count, handleLogin, index, isLogin, title, onNextSlide,
               <Text style={bottomOverlayStyles.descBold}>{' without storing\n'}</Text>
               {'your data or'}
               <Text style={bottomOverlayStyles.descBold}>{' sharing'}</Text>
-              {' it with BetterSocial'}
+              {' it with Helio'}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the text from "BetterSocial" to "Helio" seems to be a branding update. Ensure that this change is consistent across all instances in the application to maintain consistency.

```diff
- {' it with BetterSocial'}
+ {' it with Helio'}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->